airflow/www/views.py

```diff
@@ -154,6 +154,18 @@ def truncate_task_duration(task_duration):
     return int(task_duration) if task_duration > 10.0 else round(task_duration, 3)
 
 
+def sanitize_args(args: dict[str, str]) -> dict[str, str]:
+    """
+    Remove all parameters starting with `_`
```

Comment:
```suggestion
    Remove all parameters starting with `_`
    
```

Need a blank line for it to render properly

Reply:
yep